src/main/java/aquality/appium/mobile/application/Application.java

```diff
@@ -0,0 +1,97 @@
+package aquality.appium.mobile.application;
+
+import aquality.appium.mobile.configuration.IApplicationProfile;
+import aquality.selenium.core.applications.IApplication;
+import aquality.selenium.core.configurations.ITimeoutConfiguration;
+import aquality.selenium.core.localization.ILocalizedLogger;
+import io.appium.java_client.AppiumDriver;
+import org.openqa.selenium.remote.service.DriverService;
+
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class Application implements IApplication {
+
+    private final ILocalizedLogger localizedLogger;
+    private final IApplicationProfile applicationProfile;
+    private final ITimeoutConfiguration timeouts;
+    private Duration timeoutImpl;
+    private final AppiumDriver appiumDriver;
+    private final DriverService driverService;
+
+    public Application(AppiumDriver appiumDriver) {
```

Comment:
can we call constructor below like this(appiumDriver, null)

Reply:
yes we can